### PR TITLE
Touchup audio nav hover states

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Minor stylistic changes and comment about feature flag

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-touchups

File: src/containers/nav/desktop/NavAudio.module.css [hunk 1]
```diff
@@ -12,31 +25,42 @@
...
 .audio.show .earnAudio:hover {
-  cursor: pointer;
+  transform: scale(1.03);
 }
 
 .audio.show .earnAudio:active {
   transform: scale(.97);
 }
 
+.audio.hasBalance {
```
Comment (sddioulde) [inline src/containers/nav/desktop/NavAudio.module.css:35]: nice

**Review by sddioulde — APPROVED** [review]

File: src/containers/nav/desktop/NavAudio.tsx [hunk 2]
```diff
@@ -37,11 +39,14 @@ const NavAudio = () => {
   const audioBadge = audioTierMapPng[tier]
 
   if (!isEnabled || !account) {
-    return null
+    return <div className={styles.audio} />
```
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply src/containers/nav/desktop/NavAudio.tsx]: Yeah -- ok this is a super good point. Left nav just isn't a good place for feature flags. Undid this change, but we should relaunch this with the feature flag removed ASAP

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-touchups

**Pull request merged**